libpermian/caserunconfiguration/__init__.py

```diff
@@ -180,6 +180,35 @@ def openLogfile(self, name, mode="r", autoadd=False):
             self.addLog(name, log_path)
         return log_fo
 
+    def dropFile(self, name, data):
+        """
+        Create file *name* (relative to crcid's log path) and write
+        *data* bytes to it.
+
+        This uses same file path structure as openLogfile(), but unlike
+        openLogfile(), this will simply drop the bytes as-is into the
+        file associated with this crcid.  Also this method will not add
+        '.txt' extension to file name.
+
+            crc.dropFile("foo.fil", "I am the content of foo.fil!".encode())
+
+        If the file already exists, it is rewritten silently with the new
+        content.
+
+        Log file is automatically added to crc using *addLog()* method.
+        """
+        log_path = os.path.join(
+            self.testrun.settings.get('workflows', 'local_logs_dir'),
+            self.id,
+            name,
+        )
+        if log_path.startswith('file://'):
```

Comment:
This should not happen in this case as the "file://" protocol as the only valid way how the "file://" should be able to appear is from the `log_path = self.logs[name]` which is (correctly) not used in this implementation.

Reply:
Thing is, `log_path` will actually start with whatever is specified in 'local_logs_dir' in the INI.